src/main/antora/modules/ROOT/pages/jdbc/template.adoc

```diff
@@ -0,0 +1,48 @@
+[[jdbc.template]]
+= Template API
+
+Spring Data JDBC offers the javadoc:org.springframework.data.jdbc.core.JdbcAggregateTemplate[] as a more direct means to load and persist entities in a relational database.
+Repositories use to a large extend the `JdbcAggregateTemplate` to implement their features.
```

Comment:
Change to: 

To a large extent, repositories use `JdbcAggregateTemplate` to implement their features.